metadata.rb

```diff
@@ -1,16 +1,17 @@
 name 'osquery'
 maintainer 'Jack Naglieri'
```

Comment:
Are we planning to submit these changes back upstream? If not, maybe we should update the maintainer info here in our fork

Reply:
Eventually I would like to add back to the original project but it seems pretty dead and the reason I forked it. I can update the maintainer and get the advice regarding Copyright

Reply:
updated copyright on modification made